### PR TITLE
GE_DMI3_Torso dataset

### PR DESCRIPTION
**Pull request by KrisThielemans**
Files for GE_DMI3_Torso. I also made an attempt to adjust the examples to cope with non-TOF multfactors for TOF data, but there are remaining problems in the BSREM set-up, so instead, I've chosen to create the data with TOF multfactors as well.

### CODE REVIEW

**Comment by KrisThielemans** [comment]
@casperdcl could be interesting to add this to our test runs in the leaderboard

**Comment by KrisThielemans** [comment]
Data is on https://petric.tomography.stfc.ac.uk/data/GE_DMI3_Torso/

File: main_OSEM.py [hunk 1]
```diff
@@ -63,9 +63,16 @@ def __init__(self, data: Dataset, num_subsets: int = 7, update_objective_interva
 
             acquisition_model_subset = STIR.AcquisitionModelUsingParallelproj()
             acquisition_model_subset.set_additive_term(additive_term_subset)
-            acquisition_model_subset.set_up(prompts_subset, self.x)
+            if prompts_subset.dimensions()[0] > 1 and multiplicative_factors_subset.dimensions()[0] == 1:
+                # non-TOF multiplicative factors for TOF data
```
Comment (casperdcl) [inline main_OSEM.py:67]: Oh is this a backwards-incompatible change? Current participants won't be aware of this.

**Comment by KrisThielemans** [comment]
@casperdcl I've removed the commits trying to cope with nonTOF multfactors for TOF data (and moved it to #114). This should now be to your liking 😄 .

File: main_OSEM.py (hunk 1 above)
Comment (KrisThielemans) [reply main_OSEM.py]: removed this now.

**Comment by casperdcl** [comment]
> @casperdcl could be interesting to add this to our test runs in the leaderboard

added to #91...

**Pull request merged**